tests/test_angles.py

```diff
@@ -43,3 +44,10 @@ def test_generator():
     a = md.compute_angles(ptraj, triplets)
     b = md.compute_angles(ptraj, triplets2)
     eq(a, b)
+
+def test_angle_180(get_fn):
+    # Test that linear coordinates yield 180 degrees, not "nan"
+    a = md.load(get_fn('mol.gro'))
+    b = md.compute_angles(a, [[0, 1, 2]])
```

Comment:
```suggestion
@pytest.mark.parametrize("optimize", [True, False])
def test_angle_180(get_fn, optimize):
    # Test that linear coordinates yield 180 degrees, not "nan"
    a = md.load(get_fn('mol.gro'))
    b = md.compute_angles(a, [[0, 1, 2]], opt=optimize)
```

IIRC these functions have "NumPy" and "our own C code" pathways - as you modified both - so in principle both paths should be tested

Reply:
Ah - now we get failures due to the default tolerance in pytest.approx (1e-6) being exceeded. If I reduce it to 1e-4, things are OK again. Are you happy to accept this?